pkg/applicationserver/io/packages/packages.go

```diff
@@ -130,7 +131,14 @@ func (s *server) handleUp(ctx context.Context, msg *ttnpb.ApplicationUp) error {
 			ctx := log.NewContextWithField(ctx, "package", name)
 			err := handler.HandleUp(ctx, pair.defaultAssociation, pair.association, msg)
 			if err != nil {
+				errorLabel := unknown
+				if ttnErr, ok := errors.From(err); ok {
+					errorLabel = ttnErr.FullName()
+				}
+				packagesMetrics.messagesFailed.WithLabelValues(name, errorLabel).Inc()
```

Comment:
I feel like this belongs in a `registerXXX()` in `observability.go` to not clutter the codebase here